blogs/network_events/index.md

```diff
@@ -0,0 +1,205 @@
+
+Example of admin network policy and egress network policy to show multiple events use case :
+
+```yaml
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: allow-egress-iperf
+spec:
+  egress:
+  - action: Pass
+    to:
+    - namespaces: {}
+  priority: 10
+  subject:
+    namespaces:
+      matchLabels:
+        kubernetes.io/metadata.name: iperf
+```
+
+![admin network policy events](./images/admin_policy_network_events.png)
+
+3. Observe Egress Firewall Traffic
+
+For egress firewall monitoring, eBPF can:
+
+Trace Egress Traffic: Track outbound traffic and see whether it adheres to egress firewall rules.
+Detect Unauthorized Outbound Connections: You can flag outbound traffic that violates egress rules.
+
+Example includes three events, one for network policy, another for admin network policy and the third for egress firewall
+
+```yaml
+apiVersion: k8s.ovn.org/v1
+kind: EgressFirewall
+metadata:
+  name: default
+  namespace: iperf
+spec:
+  egress:
+  - to:
+      cidrSelector: 8.8.8.8/24
+    type: Allow
+  - to:
+      cidrSelector: 0.0.0.0/0
+    type: Deny
+```
+
+![egress firewall network events](./images/egress_fw_network_events.png)
+
+## Integration with Network Observability Stack
+
+- `eBPF Agent`:
+
+The agent will manage a new kprobe hook through a configuration option.
+It will also register with the OVN observability library, allowing eBPF-generated events to be enriched
+into meaningful strings.
+
+- `Agent User Space`:
+
+User space will handle the aggregation of events for processing.
+
+- `Flowlogs Pipeline`:
+
+Generated flows will be transmitted over gRPC to the FlowLog pipeline,
+where the flow content will be enriched with Kubernetes metadata.
+
+- `Loki storage and Console-plugin`:
+
+The enriched flows will then be sent to Loki storage, making them available for
+consumption by the NetObserv console plugin. The new events will be displayed in the console.
+
+- `New Filtering Capability`:
+
+A filter is introduced to filter flows based on network event enrichments.
+
+![network events building blocks](./images/network_events_design.drawio.png)
+
+## Configuration
+
+- Enable `OVNObservability` featureGate
+
+```shell
+oc edit featuregates.config.openshift.io cluster
+```
+
+```yaml
+enabled:
+- name: OVNObservability
+```
+
+Note:
+ The above feature Gate might be on by default at the time this feature becomes GA.
+
+- Enable Network events in Network Observability
+
+```yaml
+sampling: 1
+privileged: true
+features:
+- "NetworkEvents"
+```
+
+Note:
+
+- in the above example we set `sampling` to 1 to be able to capture all network events without missing any.
+
+- `privileged` is set to true because `OVN observability` library needs to access local OVS socket and OVN Databases.
+
+## Availability
+
+ Networking events monitoring feature will be available as dev-preview for OCP 4.17 z-strean and Netobserv release 1.7, and will be
```

Comment:
```suggestion
The networking events monitoring feature will be available as Developer Preview for OCP 4.17 z-strean and NetObserv release 1.7, and is planned for
```

Reply:
I have a `yikes` caution against promising future features 😬 we don't do it in product docs, per https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future.

Reply:
Suggest using `planned` to make it less of a promise.